Expensify/App
# Correctly highlight chats in the chat switcher if they are unread

<Assign reviewers that know the area or changes well. Feel free to tag any additional reviewers you see fit.>

While we were correctly highlighting report-type chats with unread comments in the chat Switcher, we were not doing the same with user chats. This should fix that.

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/143652

### Tests
- Open two different instances of Nativa chat and log in with two different accounts (You can open one in incognito mode, for example).
- Open a chat with someone else in Account 1
- Send a message from Account 2 to Account 1. You should get the notification and see the chat highlighted in the LHN. Do not open it.
- Go to chat Switcher and start searching for Account 2. When it shows up, it should do so highlighted:
<img width="298" alt="Screen Shot 2020-11-19 at 1 02 51 PM" src="https://user-images.githubusercontent.com/19366280/99663858-8ec1dd80-2a67-11eb-9e82-b55c7cd40626.png">
